packages/library-legacy-sham/src/transaction.ts

```diff
@@ -55,7 +55,7 @@ export class Transaction {
                     'us know: https://github.com/solana-labs/solana-web3.js/issues/new/choose',
             );
         }
-        const byteArray = getTransactionEncoder().encode(this.#tx as CompilableTransaction);
+        const byteArray = getTransactionEncoder().encode(this.#tx as CompilableTransaction) as Uint8Array;
```

Comment:
LOL.

Also, I should really delete this after you land this so that you never have to spend time on it again.